index.js

```diff
@@ -18,9 +19,11 @@ module.exports = function (opts) {
 			return;
 		}
 
+		var target = (opts.abspath) ? path.relative(opts.abspath, file.path) : file.path;
```

Comment:
I'd rather see this fixed in `vulcanize`. Expecting different main input based on an option is such an anti-pattern.